src/Application/Actions/GetPaymentMethods.php

```diff
@@ -41,22 +41,22 @@ protected function action(Request $request, Response $response, array $args): Re
 
         $regularGivingPaymentMethod = null;
 
-        // exclude payment methods with 'allow_redisplay' set to limited:
-        $displayableMethods = array_values(array_filter(
+        $nonRegularGivingMethods = array_values(array_filter(
             $paymentMethodArray,
             static function (array $paymentMethod) use ($donor, &$regularGivingPaymentMethod) {
                 if ($paymentMethod['id'] === $donor->getRegularGivingPaymentMethod()?->stripePaymentMethodId) {
                     $regularGivingPaymentMethod = $paymentMethod;
```

Comment:
this had an implicit return null before - replaced it with explicit return false.

Reply:
Does this mean we're removing the Regular Giving method from the cards you manage in the account page?

Reply:
It's on the page, its just shown by itself instead as part of the main list. That's why line 49 assigns $regularGivingPaymentMethod before returning null. This PR isn't intending to change that.

Reply:
I can't 100% remember why we decided to only list it separately, but maybe we thought it would be confusing having it twice, and there are special rules for deleting it - I think you can't delete it, you can only replace it with another method, unless you have zero active mandates.

Reply:
Right, yep that does now ring a bell. I guess the mutually exclusive on- or off-session choice for the different purposes also makes it potentially clearer to keep them distinct.